makefiles/rules.mk

```diff
@@ -34,7 +34,7 @@ $(THEOS_BUILD_DIR):
 endif
 
 $(THEOS_OBJ_DIR):
-	@cd $(THEOS_BUILD_DIR); mkdir -p $(THEOS_OBJ_DIR_NAME)
+	@cd $(_THEOS_LOCAL_DATA_DIR); mkdir -p $(THEOS_OBJ_DIR_NAME)
```

Comment:
Just `@mkdir -p $(THEOS_OBJ_DIR)` should be enough.

![](https://usercontent.irccloud-cdn.com/file/zDVknA8x/image.png)